### PR TITLE
Integration tests for snow stage

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added new automated tests to verify correctness of my new code.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [x] I've described my changes in the section below.

### Changes description
Added integration tests for snow stage

### CODE REVIEW

File: tests_integration/test_stage.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+from tests_integration.snowflake_connector import snowflake_session
+from tests_integration.test_utils import (
+    extract_row_from_mock,
+    extract_row_from_snowflake_session,
+    contains_row_with,
+    not_contains_row_with,
+)
+
+# TODO add database when we will have one
+DATABASE_NAME = ""
```
Comment (sfc-gh-turbaszek) [inline tests_integration/test_stage.py:15]: Shoudl we generate this as uuid instead of empty one?

File: tests_integration/test_stage.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        assert contains_row_with(
+            extract_row_from_mock(mock_print),
+            {"source": filename, "target": filename, "status": "UPLOADED"},
+        )
+
+    runner.invoke_with_config(["stage", "list", stage_name])
+    result = snowflake_session.execute_string(
+        f"use database {DATABASE_NAME}; list @{stage_name}"
+    )
+    assert extract_row_from_mock(mock_print) == extract_row_from_snowflake_session(
```
Comment (sfc-gh-turbaszek) [inline tests_integration/test_stage.py:49]: Maybe we can dump the `extract` prefix, wdyt?

Comment (sfc-gh-astus) [reply tests_integration/test_stage.py:49]: I agree 👍

File: tests_integration/test_stage.py (hunk 1 above)
Comment (sfc-gh-astus) [reply tests_integration/test_stage.py:15]: We will but we have other task for it

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**